tests/test_app.py

```diff
@@ -42,7 +43,7 @@ def json(self):
 
 class FakeRequest:
     def __init__(self, status_code: int, text: str):
-        self.status_code = status_code
+        self.status_code = status_code0
```

Comment:
Hah I missed that one. derp.

On 9 February 2016 at 16:45, Rafael Carício <notifications@github.com>
wrote:

> In tests/test_app.py
> <https://github.com/zalando/lizzy/pull/45#discussion_r52325249>:
>
> > @@ -42,7 +43,7 @@ def json(self):
> >
> >  class FakeRequest:
> >      def __init__(self, status_code: int, text: str):
> > -        self.status_code = status_code
> > +        self.status_code = status_code0
>
> Typo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/lizzy/pull/45/files#r52325249>.
>



-- 
Daniël Franke